ai/main.py

```
@@ -31,4 +45,4 @@ def image(book_id: int,page_number: int, authorization: str = Header(None)):
         else:
             raise HTTPException(status_code=404,detail="데이터가 없습니다.")
     except Exception as e:
-        raise HTTPException(status_code=500,detail=f"DB 연결 실패: {str(e)}")
\ No newline at end of file
+        raise HTTPException(status_code=500,detail=f"DB 연결 실패: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)